package.json

```diff
@@ -422,6 +422,24 @@
       "isFramework": false,
       "integrity": "9944EBD6EE06BD595BCADD3057CD9BEF4105C3A3952DAE03E54F3114E2E6661F"
     },
+    {
+      "id": "RoslynCopilot",
+      "description": "Language server for Roslyn Copilot integration (Windows)",
+      "url": "https://roslynomnisharp.blob.core.windows.net/releases/1.39.12/omnisharp-linux-musl-arm64-net6.0-1.39.12.zip",
+      "installPath": ".roslyncopilot",
+      "platforms": [
```

Comment:
if there is a better way of specifying support for all platforms and architectures the extension supports, I'd love to put that in so this list is a little more maintainable.

Reply:
We could introduce the concept of "neutral" which would be a catch-all. Are there no plans to generate R2R builds of these libraries?

Reply:
It is used by the [PackageManager](https://github.com/dotnet/vscode-csharp/tree/main/src/packageManager) functionality which builds out of our repo. In particular we would update [our package filter](https://github.com/dotnet/vscode-csharp/blob/main/src/packageManager/packageFilterer.ts#L18) to also include these "neutral" packages

Reply:
Cool, gotcha. As long as that's the only place that needs a change, happy to make that update. Thought this was consumed by other tooling. Thanks for the confirmation 👍